web/src/main/webapp/my-app/layout/partials/remove-button.html

```diff
@@ -18,9 +18,11 @@
     under the License.
 
 -->
-<md-button ng-controller="RemoveWidgetController as removeCtrl" class="widget-action widget-remove md-icon-button"
-           aria-label="remove {{ widget.fname }} widget from your home screen"
-           ng-click="removeCtrl.removeWidget(widget)"
-           ng-hide="GuestMode">
-  <md-icon>close</md-icon>
-</md-button>
+<md-menu-item>
+  <md-button ng-controller="RemoveWidgetController as removeCtrl"  class="widget-remove"
+             aria-label="remove {{ widget.fname }} widget from your home screen"
+             ng-click="removeCtrl.removeWidget(widget)"
+             ng-hide="GuestMode">
+    <md-icon>delete_forever</md-icon><span>Remove from home</span>
```

Comment:
Given that the action is undo-able and even if not immediately un-done the user can typically find the item again in the app directory and again add it to their home, maybe [regular `delete` rather than `delete_forever`](https://material.io/tools/icons/?search=delete)?